README.md

```diff
@@ -1208,13 +1208,19 @@ const response = await replicate.request(route, parameters);
 
 | name                 | type   | description                                                  |
 | -------------------- | ------ | ------------------------------------------------------------ |
-| `options.route`      | string | Required. REST API endpoint path.                            |
-| `options.parameters` | object | URL, query, and request body parameters for the given route. |
+| `options.route`      | `string` | Required. REST API endpoint path.                            |
+| `options.params` | `object` | URL query parameters for the given route. |
+| `options.method` | `string` | HTTP method for the given route. |
+| `options.headers` | `object` | Additional HTTP headers for the given route. |
+| `options.data` | `object | FormData` | Request body. |
+| `options.signal` | `AbortSignal`  | Optional `AbortSignal`. |
```

Comment:
Can you fix the formatting here so the table's dividers align vertically, please?